pkg/processor/procbuiltin/unwrap.go

```diff
@@ -0,0 +1,310 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package procbuiltin
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/conduitio/conduit/pkg/foundation/cerrors"
+	"github.com/conduitio/conduit/pkg/foundation/log"
+	"github.com/conduitio/conduit/pkg/foundation/multierror"
+	"github.com/conduitio/conduit/pkg/inspector"
+	"github.com/conduitio/conduit/pkg/processor"
+	"github.com/conduitio/conduit/pkg/record"
+	"github.com/rs/zerolog"
+)
+
+type unwrapProcessor struct {
+	unWrapper unWrapper
+
+	inInsp  *inspector.Inspector
+	outInsp *inspector.Inspector
+}
+
+// unWrapper unwraps the formatted record from the openCDC record
+type unWrapper interface {
```

Comment:
Unwrap is a single word, I think we should use `unwrapper` instead of `unWrapper`.

Reply:
goLand complained about it not being a word, so this one had no squiggly lines under it, that's why I renamed it. But I agree that it should be `unwrapper`

Reply:
Time to update the dictionary, Goland! 🤓